### PR TITLE
Update-README.md

### PR DESCRIPTION
**Pull request by murakamiharuto**
I have just changed the Readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-## MERN Authentication
+### MERN Authentication
 
-## How to Use
```
Comment (max-dev001) [inline README.md:3]: well done.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+###How to Use
 
 - Clone the repository.
```
Comment (max-dev001) [inline README.md:5]: It is as origin.

File: README.md [hunk 3]
```diff
@@ -10,7 +10,7 @@
 - Run command **npm install** to install the dependencies.
```
Comment (max-dev001) [inline README.md:10]: well.

File: README.md [hunk 4]
```diff
@@ -10,7 +10,7 @@
...
 - Run command **npm run server** to start the server.
```
Comment (max-dev001) [inline README.md:11]: well.

File: README.md [hunk 5]
```diff
@@ -10,7 +10,7 @@
...
 
-## Screenshots
+### Screenshots
 
 ![Register Screen](https://github.com/AnumMujahid/mern-authentication/blob/main/Resources/a1.png)
```
Comment (max-dev001) [inline README.md:15]: Okay.

**Review by max-dev001 — COMMENTED** [review]
Well done.

**Comment by murakamiharuto** [comment]
perfect!!!!!!

**Pull request merged**